components/layout_2020/flow/mod.rs

```diff
@@ -332,10 +311,9 @@ impl BlockLevelBox {
     }
 }
 
-#[derive(PartialEq)]
-enum BlockLevelKind {
-    SameFormattingContextBlock,
-    EstablishesAnIndependentFormattingContext,
+enum BlockLevelKind<'a> {
```

Comment:
Would it make more sense to name that `BlockKind`?

Reply:
Or `BlockContents`, or something with "contents" somewhere. I don't like how it has less variants than `BlockLevelBox`.

Reply:
Done.